packages/coil-extension/src/background/consts/extensionInstances.ts

```diff
@@ -0,0 +1,36 @@
+// add the id's of the extensions to check their active states
+export interface ExtensionInstance {
+  name: string
+  id: string
+}
+
+export type BrowserType = 'chrome' | 'firefox' | 'edge'
+
+export const EXTENSIONS: Record<BrowserType, Array<ExtensionInstance>> = {
+  chrome: [
+    {
+      name: 'CoilDevDist2',
```

Comment:
We forgot the main chrome one:
https://chrome.google.com/webstore/detail/coil/locbifcbeldmnphbgkdigjmkbfkhbnca?hl=en